frame/staking/src/pallet/mod.rs

```diff
@@ -826,7 +826,7 @@ pub mod pallet {
 
 		#[cfg(feature = "try-runtime")]
 		fn try_state(n: BlockNumberFor<T>) -> Result<(), &'static str> {
-			Self::do_try_state(n)
+			Self::do_try_state(n, TestMode::Fast)
```

Comment:
Maybe we should support passing data into `try_state` so that we can run the full tests manually without code change.

Reply:
Yeah I think we should look into how we want to do this across pallets and have a way to run full test manually probably as an arg to on-runtime-upgrade instead of having to change code.